public/app/features/manage-dashboards/types.ts

```diff
@@ -17,3 +17,11 @@ export type DeleteDashboardResponse = {
   message: string;
   title: string;
 };
+
+export interface ListPublicDashboardResponse {
```

Comment:
Hmm. not sure if this is the rate place to put this. I sort of like leaving it next to the place that is leaving it. You are correct though, this is management, but it doesn't seem like this file is used much?

Reply:
we're using it in 3 different places and should be inside an interface folder or type file

Reply:
if it were just used inside the component and anywhere else, I would put it inside the component file

Reply:
Maybe put it in the API file?

Reply:
I would leave it here